src/server/game/Entities/Creature/CreatureTemplate.cpp

```diff
@@ -0,0 +1,171 @@
+/*
+* Copyright (C) 2008-2018 TrinityCore <https://www.trinitycore.org/>
+* Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
```

Comment:
Lines 2 through 17 should have 1 space indentation for all the Copyright headers.
I presume the code is unchanged (copied from other files), for this MaNGOS line.